app/plugin.go

```diff
@@ -165,29 +166,36 @@ func (a *App) InitPlugins(pluginDir, webappPluginDir string) {
 
 	prepackagedPluginsDir, found := fileutils.FindDir("prepackaged_plugins")
 	if found {
-		if err := filepath.Walk(prepackagedPluginsDir, func(walkPath string, info os.FileInfo, err error) error {
-			if !strings.HasSuffix(walkPath, ".tar.gz") {
-				return nil
+		var walkPaths []string
+		filepath.Walk(prepackagedPluginsDir, func(walkPath string, info os.FileInfo, err error) error {
```

Comment:
Can we still handle the potential `err` here?

Reply:
Done.